webapp/src/main/webapp/test/output.test.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Unit tests for output.js
+ */
+describe('Output', function() {
+
+  // inject the HTML fixture for the tests
+  beforeEach(function() {
+    var fixture = '<div id="fixture"><div id="warning"></div></div>';
+
+    document.body.insertAdjacentHTML(
+      'afterbegin', 
+      fixture);
```

Comment:
Could you please explain what this does? What's the 'afterbegin' part?

Reply:
Following http://www.bradoncode.com/blog/2015/02/27/karma-tutorial/